### PR TITLE
added an endpoint to profitability index using annual cash flows

### PR DESCRIPTION
**Pull request by ShamanthVallem**
For issue #146 
changes made:
1) Created an endpoint to calculate profitability endpoint using annual cash flows

### CODE REVIEW

**Comment by nikhil25803** [comment]
Hey @ShamanthVallem, there are some merge conflicts, please do resolve them first,

**Comment by ShamanthVallem** [comment]
@nikhil25803 I resolved all merge conflicts.

File: ENDPOINTS.md [hunk 1]
```diff
@@ -899,6 +899,20 @@
...
+**GET** `/profitability_index2`
+- Required parameters : `initial_investment`,`annual_cash_flows`, `discount_rate`
+- Note: annual_cash_flows should be space seperated
+- Sample Output
+```py
+{
+   "Tag" : "Profitability Index",
+    "initial_investment" : "$10000",
+    "annual_cash_flows" : "5000 4000 3000",
+    "discount_rate" : 10%,
```
Comment (nikhil25803) [inline ENDPOINTS.md:911]: Case the `discount_rate` into a string, and `profitability_index` without any dollar or sign.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]
Requested some changes, please look into it.

**Comment by ShamanthVallem** [comment]
@nikhil25803 Points were not awarded for this PR, can u please check

**Comment by nikhil25803** [comment]
> @nikhil25803 I did not receive points for this Pull request, can u please check

Codepeak is wrapped up now, no points will be reflected in the leaderboard anymore.

**Pull request merged**